src/pages/examples/cedar-chart.hbs

```diff
@@ -0,0 +1,126 @@
+---
+title: Dynamic feature charting with Cedar
+description: Plot feature attributes on a dynamic chart that updates as users pan and zoom. This demo relies on <a href="https://github.com/Esri/cedar">Esri Cedar</a> to render an interactive scatterplot.
+layout: example.hbs
+---
+<!-- Load Cedar files from CDN -->
+<!-- load the amCharts base library -->
+<script src="https://www.amcharts.com/lib/3/amcharts.js"></script>
+<!-- for scatter and bubble charts -->
+<script src="https://www.amcharts.com/lib/3/xy.js"></script>
+<!-- load the arcgis-rest-js scripts -->
+<script src="https://unpkg.com/@esri/arcgis-rest-request"></script>
+<script src="https://unpkg.com/@esri/arcgis-rest-feature-service"></script>
+<!-- optionally load an amcharts theme; cedar provides a calcite theme -->
+{{!-- <script src="https://unpkg.com/@esri/cedar/dist/umd/themes/amCharts/calcite.js"></script> --}}
+<!-- load cedar -->
+<script src="https://unpkg.com/@esri/cedar/dist/umd/cedar.js"></script>
+
+<style>
+  .chart-container {
+    height: 245px;
+    width: 245px;
+  }
+
+  #info-pane {
+    position: absolute;
+    top: 10px;
+    right: 10px;
+    z-index: 400;
+    background: white;
+  }
+
+  .amcharts-chart-div > a {
+    display: none !important;
+  }
+
+  #chart {
+    height: 100%;
+  }
+</style>
+
+<div id="map"></div>
+<div id="info-pane" class="leaflet-bar chart-container">
+  <div id="chart" ></div>
+</div>
+
+<script>
+  // STEP 1: MAKE A MAP AND ADD LAYERS
+  var map = L.map('map').setView([45.526, -122.667], 13);
+
+  L.esri.basemapLayer('Topographic').addTo(map);
+
+  // create and add a feature layer
+  // features from this layer will appear in the Chart.js scatterplot
+  var treesFeatureLayer = L.esri.featureLayer({
+    url: 'https://services.arcgis.com/rOo16HdIMeOBI4Mb/arcgis/rest/services/Heritage_Trees_Portland/FeatureServer/0',
+  });
+
+  treesFeatureLayer.addTo(map);
+
+  // STEP 2: DEFINE A CHART
+  var chart;
+  treesFeatureLayer.on('load', function() {
```

Comment:
Could `once` be used here to replace the `on` and subsequent `off` below?  Just an idea...not a big deal to me either way.

https://leafletjs.com/reference-1.4.0.html#evented-once

```suggestion
  treesFeatureLayer.once('load', function() {
```